pkg/services/ngalert/provisioning/contactpoints.go

```diff
@@ -88,13 +88,14 @@ func (ecp *ContactPointService) GetContactPoints(ctx context.Context, q ContactP
 		}
 	}
 
-	var contactPoints []apimodels.EmbeddedContactPoint
-	for _, gr := range grafanaReceivers {
+	contactPoints := make([]apimodels.EmbeddedContactPoint, len(grafanaReceivers))
+	for i, gr := range grafanaReceivers {
 		contactPoint, err := GettableGrafanaReceiverToEmbeddedContactPoint(gr)
 		if err != nil {
 			return nil, err
 		}
-		contactPoints = append(contactPoints, contactPoint)
+
+		contactPoints[i] = contactPoint
```

Comment:
shouldn't we stick with append here?

Reply:
I could go either way - since it's only possible to populate `contactPoints` with exactly the same number of elements as `grafanaReceivers` I thought this would be slightly more explicit.

Reply:
fair enough 👍